clang/test/SemaSYCL/intel-fpga-local.cpp

```diff
@@ -110,6 +110,7 @@ void check_ast()
   [[intel::fpga_memory("MLAB")]] unsigned int doublepump_mlab[64];
 
   //CHECK: VarDecl{{.*}}max_replicates
+  //CHECK: IntelFPGAMemoryAttr{{.*}}Implicit
```

Comment:
Just a minor nit:  please add a comment at the top of the test to indicate what this does.  Thanks!

Reply:
Done